terraform/modules/services/fat-buyer-ui/ecs.tf

```diff
@@ -111,6 +111,14 @@ resource "aws_ecs_task_definition" "fat_buyer_ui" {
           {
           "name": "GUIDED_MATCH_SERVICE_ROOT_URL",
           "value": "${var.api_invoke_url}"
+          },
+          {
+          "name": "AGREEMENTS_SERVICE_API_KEY",
```

Comment:
Do you think it's worth passing this in via container secrets? Little point I guess unless you can restrict access to view the keys in the AWS Console too - which I can't recall atm.. 🤔

Reply:
@tommyb82   I did think of that, but as you say if you have console access - then these keys are all visible via the API Gateway anyway so wasn't sure if that added any value. I can add it to the tech debt register to look at again?

Reply:
Actually - I can give it a quick try on SBX5 to see if it still works - makes sense to do it still I think